apps/backend/src/schema/schema.ts

```diff
@@ -0,0 +1,52 @@
+import { z } from 'zod';
+
+const addressSchema = z.object({
+    address: z.string().min(1),
+    city: z.string().min(1),
+    state: z.string().min(1),
+    zipcode: z.number(),
+})
+
+// Regex to check for phone numbers formatted with `-`, ' ', and '.' separators, 
+// and with and without area code surrounded by parenthesis
+const phoneNumberRegex = RegExp('`^(\+\d{1,2}\s)?\(?\d{3}\)?[\s.-]\d{3}[\s.-]\d{4}$`')
+const phoneNumber = z.string().regex(phoneNumberRegex)
+
+// Part of form to be filled out by the child's parent/legal guardian
+const guardianFormSchema = z.object({
+    childsName: z.string().min(1),
+    ssn: z.number(),
+    dob: z.date(),
```

Comment:
were you able to test this? from the [documentation](https://zod.dev/?id=dates), it looks like this requires a `Date` instance, a string wouldn't pass. Or you could try using `z.coerce.date()`?

Reply:
Yes, this worked when I passed in a string in this format '"2012-04-23T18:25:43.511Z"'

Reply:
hmm nit but that seems like an unwieldy API for the frontend to use... I'd still prefer breaking a date down into month, day, year but I'll leave it up to you

Reply:
The frontend will just be able to pass a Date object in. I think that will be easier than separating out the month, day, and year.

Reply:
sounds good, go ahead and merge!